src/js/populate_question_form.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+/* global wordle_globals */
+
+var inverted_wordles = {};
+
+// Populate the data on the answer question page
+inverted_wordles.setFormData = function (response, options) {
+    response.json().then(questionFile => {
+        inverted_wordles.setEscapedContent(options.selectors.questionTitle, questionFile.title);
+        inverted_wordles.setEscapedContent(options.selectors.entryMaxLength, questionFile.entryMaxLength);
+
+        let entriesHtml = "";
+        for (let i = 0; i < questionFile.entries; i++) {
+            entriesHtml += "<input type=\"text\" autocomplete=\"false\" ";
```

Comment:
This looks like a good use case for an ES6 template together with a use of questionFile.entries.map((entry, i) => ....